image_client/picturae_csv_create.py

```diff
@@ -246,7 +249,7 @@ def taxon_concat(self, row):
                     hybrid_base = full_name
                     full_name = taxon_strings[0]
                     if full_name != genus:
-                        full_name = taxon_strings[-1]
+                        full_name = " ".join(taxon_strings[:2])
```

Comment:
changed this line to just check the entire genus string for genus level hybrids , this tends to work better for TNRS and also for clearer troubleshooting on picturae_unmatch_tab.